tests/operators/test_agnostic_aggregate_check.py

```diff
@@ -79,6 +82,16 @@ def setUpClass(cls):
     def clear_run(self):
         self.run = False
 
+    def get_dag(self):
+        very_high_number = 99999999
+        return DAG(
+            "test_dag_" + str(random.randint(0, very_high_number)),
+            default_args={
+                "owner": "airflow",
+                "start_date": DEFAULT_DATE,
+            },
+        )
+
```

Comment:
So instead of using this, you should look into [pytest fixtures](https://docs.pytest.org/en/6.2.x/fixture.html).

If you look [here](https://github.com/astro-projects/astro/blob/main/conftest.py#L36-L41), you'll see that we have a fixture that generates a DAG AND deletes it once the test is completed. I want us to move towards pytest fixtures so you should read up on these.

Reply:
Done